### PR TITLE
Practical

### PR DESCRIPTION
**Pull request by Akshay8122**
This is all about my practical and i have learn many things to this projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 I see merge conflict marker here. Resolve this.

File: index.html [hunk 2]
```diff
@@ -9,4 +10,355 @@
...
+    <title>Flipcart home page</title>
+
+</head>
+
+<body>
+
+    <!--Navbar-->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top py-1">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#"><img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/f.png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 Absolute path for assets like **/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/f.png** will work only on your PC. Add assets to project and replace this with relative path.

File: index.html [hunk 3]
```diff
@@ -9,4 +10,355 @@
...
+            </div>
+        </section>
+        <!--End Books and Musical section -->
+
+    </div>
+
+
+</body>
+
+>>>>>>> practical
```
Comment (harshilpradhan-simformsolutions) [inline index.html:363]: @Akshay8122 Merge conflict marker

File: style.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
+/*General*/
+*{
+    margin: 0px;
+
```
Comment (harshilpradhan-simformsolutions) [inline style.css:2]: Remove this empty line

File: style.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    border-radius: none;
+}
+.value{
+    border: none;
+    background: none;
+    margin:0;
+}
+
+.m1{
+
```
Comment (harshilpradhan-simformsolutions) [inline style.css]: Remove this empty line

File: style.css [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+.m1{
+    
+    border:none;
+    background: none;
+    margin-left: 195px;
+    position: relative;
+}
+.btn{
+
```
Comment (harshilpradhan-simformsolutions) [inline style.css]: Remove this empty line

File: style.css [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    
+}
+.nav-link{
+  margin-left: 15px;
+  color: aliceblue;
+ 
+}
+
+.b1{
+border: none;
```
Comment (harshilpradhan-simformsolutions) [inline style.css]: Add proper spacing to CSS properties in .b1 class

File: style.css [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+width: 100px;
+height: 35px;
+margin-left: 125px;
+float: right;
+color: blue;
+}
+
+/*Deals of the day*/
+.light{
+color: lightslategray;
```
Comment (harshilpradhan-simformsolutions) [inline style.css]: Add proper spacing in line no. 76 and 78

File: style.css [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+opacity:0.2;
+}
+
+.text{
+    font-size: 25px;
+    margin-left:5px;
+}
+ 
+.clock{
+margin-left:20px;
```
Comment (harshilpradhan-simformsolutions) [inline style.css]: Add proper spacing

File: index.html [hunk 10]
```diff
@@ -9,4 +10,355 @@
...
+
+    <!-- Deal section-->
+    <div class="Container-main pt-lg-5">
+        <section>
+            <div class="container p-0 pt-lg-5 ml-10">
+                <div class="row">
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (8).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 11]
```diff
@@ -9,4 +10,355 @@
...
+                            </div>
+                            <h7> Top offers </h7>
+                        </div>
+                    </div>
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (7).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 12]
```diff
@@ -9,4 +10,355 @@
...
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Grocery </h7>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image.png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 13]
```diff
@@ -9,4 +10,355 @@
...
+                            </div>
+                            <h7> Mobiles </h7>
+                        </div>
+                    </div>
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (2).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 14]
```diff
@@ -9,4 +10,355 @@
...
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (5).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 15]
```diff
@@ -9,4 +10,355 @@
...
+                            </select>
+                        </div>
+                    </div>
+
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (1).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 16]
```diff
@@ -9,4 +10,355 @@
...
+                            <h7> Home </h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (3).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 17]
```diff
@@ -9,4 +10,355 @@
...
+                            <h7> Aplliances </h7>
+                            <select class="value">
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (4).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 18]
```diff
@@ -9,4 +10,355 @@
...
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Travel </h7>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (6).png"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 19]
```diff
@@ -9,4 +10,355 @@
...
+        <hr>
+        <!--Complete Deal Section-->
+
+
+        <!----Deals Of the Day-->
+        <section>
+            <div class="container">
+                <hr class="light">
+                <span class="text"> Deals Of the Day </span>
+                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/images.png" class="clock" height="2%"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 20]
```diff
@@ -9,4 +10,355 @@
...
+                    width="2%">
+                <span class="time"> 22:07:36 Left </span>
+                <button class="bk-light bg-primary">VIEW ALL</button>
+                <hr class="light">
+                <div class="inline-block">
+                    <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/card.png" class="card" />
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 21]
```diff
@@ -9,4 +10,355 @@
...
+                </div>
+            </div>
+
+
+            <div class="container p-1 pt-lg-2 ml-auto">
+                <div class="row">
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/clock.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 22]
```diff
@@ -9,4 +10,355 @@
...
+                            <p> <span class="offers">Upto 80%+Extra 10%Off</span></p>
+                            <p> Knockout Deals! </p>
+                        </div>
+                    </div>
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/home-temples.jpeg"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 23]
```diff
@@ -9,4 +10,355 @@
...
+                            <p>Space saving furniture</p>
+                        </div>
+                    </div>
+
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/tablet.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 24]
```diff
@@ -9,4 +10,355 @@
...
+                            <p> <span class="offers"> Extra 10% Off </span> </p>
+                            <p> For All Top Models </p>
+                        </div>
+                    </div>
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/decoration.jpeg"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 25]
```diff
@@ -9,4 +10,355 @@
...
+                            <p> <span class="offers"> From Rs.79 </span></p>
+                            <p> Lowest price Ever </p>
+                        </div>
+                    </div>
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/metronaut.jpeg"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 26]
```diff
@@ -9,4 +10,355 @@
...
+                            <p> <span class="offers"> From Rs.249+Extra 10%Off</span></p>
+                            <p>Jackets,dress &..</p>
+                        </div>
+                    </div>
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/lava.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 27]
```diff
@@ -9,4 +10,355 @@
...
+                <hr class="light">
+                <button class="bk-dark bg-primary">VIEW ALL</button>
+                <p> <span class="text">Books And music instruments </span> </p>
+                <p class="shop">Shop Now! </p>
+
+                <div class="row">
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/harmonium.jpeg"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 28]
```diff
@@ -9,4 +10,355 @@
...
+                            <p> <span class="offers">Upto 60% Off</span></p>
+                            <p> Shop Now! </p>
+                        </div>
+                    </div>
+
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/flutes.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 29]
```diff
@@ -9,4 +10,355 @@
...
+                            <h9>Flutes</h9>
+                            <p> <span class="offers">Upto 80% Off</span></p>
+                            <p> Shop Now! </p>
+                        </div>
+                    </div>
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/gitar1.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 30]
```diff
@@ -9,4 +10,355 @@
...
+                            <h9> Acoustic Guitars </h9>
+                            <p> <span class="offers">Strarting Rs.1799</span></p>
+                            <p> juarez,Revel. </p>
+                        </div>
+                    </div>
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/microphone.jpeg"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 31]
```diff
@@ -9,4 +10,355 @@
...
+                            <h9>Musical Accessories</h9>
+                            <p> <span class="offers">Upto 80% Off</span></p>
+                            <p> Shop Now! </p>
+                        </div>
+                    </div>
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/Musical.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html [hunk 32]
```diff
@@ -9,4 +10,355 @@
...
+                            <h9>Microphone Accessories</h9>
+                            <p> <span class="offers">Upto 80% Off</span></p>
+                            <p> Shop Now! </p>
+                        </div>
+                    </div>
+
+                    <div class="col-sm-2">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/gitar.jpeg" alt="clock"
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: Replace this with relative path

File: index.html (hunk 3 above)
Comment (harshilpradhan-simformsolutions) [inline index.html]: Merge conflict marker

File: index.html [hunk 33]
```diff
@@ -9,4 +10,355 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Flipcart home page</title>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 Spelling mistake in title tag content

File: index.html [hunk 34]
```diff
@@ -9,4 +10,355 @@
...
+
+        <!----Deals Of the Day-->
+        <section>
+            <div class="container">
+                <hr class="light">
+                <span class="text"> Deals Of the Day </span>
+                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/images.png" class="clock" height="2%"
+                    width="2%">
+                <span class="time"> 22:07:36 Left </span>
+                <button class="bk-light bg-primary">VIEW ALL</button>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 VIEW ALL button doesn't look as per the design. Check and fix.
Screenshot 2022-01-13 at 9 50 09 AM

File: index.html [hunk 35]
```diff
@@ -9,4 +10,355 @@
...
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (2).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Fashion </h7>
+                            <select class="value">
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 instead of an empty select tag, use SVG icons to show arrow in all places where applicable

**Comment by harshilpradhan-simformsolutions** [comment]
@Akshay8122 Fix all the above mentioned issues and update PR.

File: index.html (hunk 1 above)
Comment (Akshay8122) [reply index.html]: i have fixed it

File: index.html (hunk 2 above)
Comment (Akshay8122) [reply index.html]: i have fixed it

File: style.css (hunk 4 above)
Comment (Akshay8122) [reply style.css:2]: i have fixed it

File: style.css (hunk 5 above)
Comment (Akshay8122) [reply style.css]: i have fixed it

File: style.css (hunk 6 above)
Comment (Akshay8122) [reply style.css]: i have fixed it

File: style.css (hunk 7 above)
Comment (Akshay8122) [reply style.css]: i have fixed it

File: style.css (hunk 8 above)
Comment (Akshay8122) [reply style.css]: i have fixed it

File: style.css (hunk 9 above)
Comment (Akshay8122) [reply style.css]: i have fixed it

File: index.html (hunk 10 above)
Comment (Akshay8122) [reply index.html]: i have fixed it

File: index.html (hunk 11 above)
Comment (Akshay8122) [reply index.html]: i have fixed it

File: index.html (hunk 12 above)
Comment (Akshay8122) [reply index.html]: i have fixed it

**Comment by harshilpradhan-simformsolutions** [comment]
@Akshay8122 Solve merge conflicts.

**Comment by harshilpradhan-simformsolutions** [comment]
@Akshay8122 Still I see merge conflicts. solve.

Screenshot 2022-01-17 at 9 49 19 AM

Login, Cart, More options should be positioned at the right side. Check the actual image in your assignment.

Screenshot 2022-01-17 at 9 54 10 AM

Text 'Travel' doesn't look centered. Compare it with other product categories and fix.

Screenshot 2022-01-17 at 9 55 59 AM

Fix this.

**Test the app properly at your end before creating/updating a PR**

**Comment by harshilpradhan-simformsolutions** [comment]
@Akshay8122 Box shadow is not present in deals of the day section and Books and instruments section. Check with the design and fix the issue.

File: index.html [hunk 36]
```diff
@@ -224,8 +224,12 @@
       </div>
       </div>
       </div>
+<<<<<<< HEAD
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 Remove this

File: index.html [hunk 37]
```diff
@@ -224,8 +224,12 @@
...
       </section>
     
+=======
+    </section>
+>>>>>>> solve bugs
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: @Akshay8122 Remove this
Screenshot 2022-01-17 at 5 09 34 PM

Comment (Akshay8122) [reply index.html]: done

File: index.html [hunk 38]
```diff
@@ -133,183 +133,179 @@
               </h7>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h7 is not a valid html tag

File: index.html [hunk 39]
```diff
@@ -133,183 +133,179 @@
...
+
+
+          <div class="container p-1 pt-lg-0">
+            <div class="row">
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/clock.jpeg" alt="clock" height="50%" width="50%" />
+                  </div>
+                  <h9>Casio,Fastrack..</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 40]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/home-temples.jpeg" alt="clock" height="70%" width="70%" />
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/home-temples.jpeg" alt="clock" height="70%" width="70%" />
+                  </div>
+                  <h9>Home,Temples.. </h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 41]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/tablet.jpeg" alt="clock" height="70%" width="70%" />
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/tablet.jpeg" alt="clock" height="70%" width="70%" />
+                  </div>
+                  <h9>Premium Tablet...</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 42]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/decoration.jpeg" alt="clock" height="55%" width="55%" />
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/decoration.jpeg" alt="clock" height="55%" width="55%" />
+                  </div>
+                  <h9>Home decoration</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 43]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/metronaut.jpeg" alt="clock" height="50%" width="50%" />
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/metronaut.jpeg" alt="clock" height="50%" width="50%" />
+                  </div>
+                  <h9>Provogue..</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 44]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/lava.jpeg" alt="clock" height="30%" width="30%" />
+              <div class="col-sm-2">
+                <div class="well">
+                  <div class="overlay">
+                    <img src="./images/lava.jpeg" alt="clock" height="30%" width="30%" />
+                  </div>
+                  <h9>LAVA A1500</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 45]
```diff
@@ -133,183 +133,179 @@
...
+          <p class="shop">Shop Now!</p>
+          <hr class="dark" />
+
+          <div class="row">
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/harmonium.jpeg" alt="clock" height="80%" width="80%" />
+                </div>
+                <h9>Harmoniums</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 46]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/flutes.jpeg" alt="clock" height="25%" width="25%" />
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/flutes.jpeg" alt="clock" height="25%" width="25%" />
+                </div>
+                <h9>Flutes</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 47]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/gitar1.jpeg" alt="clock" height="70%" width="70%" />
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/gitar1.jpeg" alt="clock" height="70%" width="70%" />
+                </div>
+                <h9> Acoustic Guitars </h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 48]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/microphone.jpeg" alt="clock" height="43%" width="43%" />
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/microphone.jpeg" alt="clock" height="43%" width="43%" />
+                </div>
+                <h9>Musical Accessories</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 49]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/Musical.jpeg" alt="clock" height="65%" width="65%" />
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/Musical.jpeg" alt="clock" height="65%" width="65%" />
+                </div>
+                <h9>Microphone Accessories</h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

File: index.html [hunk 50]
```diff
@@ -133,183 +133,179 @@
...
-          <div class="col-sm-2">
-            <div class="well">
-              <div class="overlay">
-                <img src="./images/gitar.jpeg" alt="clock" height="50%" width="50%" />
+            <div class="col-sm-2">
+              <div class="well">
+                <div class="overlay">
+                  <img src="./images/gitar.jpeg" alt="clock" height="50%" width="50%" />
+                </div>
+                <h9> Acoustic Guitars </h9>
```
Comment (harshilpradhan-simformsolutions) [inline index.html]: h9 is not a valid html tag

**Review by harshilpradhan-simformsolutions — APPROVED** [review]
You can merge

**Pull request merged**